src/styles/layouts/_layout-about-us.scss

```diff
@@ -45,5 +45,9 @@
 
   & > footer {
     grid-area: footer;
+
+    display: flex;
+    justify-content: center;
+    box-shadow: 0 0 rem(20px) 0 box-shadow-opacity(0.16);
```

Comment:
Can we use `@mixin` to make it reusable? since I saw you applied this pattern to many classes.

Reply:
Personally, I think we don't need to make the @mixin for this since there are many different `box-shadow` properties in our website.